planning/costmap_generator/test/test_points_to_costmap.cpp

```diff
@@ -0,0 +1,215 @@
+// Copyright 2022 The Autoware Contributors
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Thank you for adding this, but unfortunately, I guess this format isn't supported by `ament_copyright`.

I'm planning to replace `ament_copyright` with `pre-commit` to support this format. (But I have no time to implement it. :pleading_face: )
https://github.com/tier4/pre-commit-hooks-ros/issues/45